recipes/fastscapelib-f2py/build.sh

```diff
@@ -1,6 +1,18 @@
 if [ $(uname) == Darwin ]; then
-    # scikit-build doesn't support setting MACOSX_DEPLOYMENT_TARGET
-    $PYTHON -m pip install . --global-option="build_ext" --global-option="--plat-name=macosx-10.9-x86_64" --no-build-isolation --no-deps --ignore-installed --no-cache-dir -vvv
+    # directly setting MACOSX_DEPLOYMENT_TARGET is not supported by scikit-build
+    $PYTHON -m pip install . \
+            --global-option="build_ext" \
+            --global-option="--plat-name=$MACOSX_DEPLOYMENT_TARGET-$OSX_ARCH" \
+            --no-build-isolation \
+            --no-deps \
+            --ignore-installed \
+            --no-cache-dir \
+            -vvv
```

Comment:
Instead of 

```
pip install . --global-option="build_ext" --global-option="--plat-name=macosx-10.9-x86_64" --no-build-isolation ... 
```

I suggest to simply use

```
pip install . --global-option="--plat-name=$MACOSX_DEPLOYMENT_TARGET-$OSX_ARCH" --no-build-isolation ... 
```

passing `build_ext` is not needed.

Reply:
Mmm [it doesn't seem to work](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=37952&view=logs&jobId=c572e5d5-7ba7-55ce-5575-7ce1d9f7161e&taskId=81967b5f-0283-5c0b-87c1-e8fdeef842a3&lineStart=239&lineEnd=239&colStart=12&colEnd=45) without passing ``build_ext`` first.

Reply:
I noticed, the equal sign should not be added. See https://github.com/conda-forge/staged-recipes/pull/8430#discussion_r287886841

Reply:
No worries!